docs/pages/machine-id/getting-started.mdx

```diff
@@ -0,0 +1,120 @@
+
+You will be using the `tbot` binary to from the package you download and install.
+
+## Step 2/4. Create Bot User
+
+Before you create a bot user, you need to determine which role you want to assign to the bot user. You can use `tctl` to examine what roles exist on your system.
+
+```bash
+$ tctl get roles --format=text
+Role    Allowed to login as                           Node Labels Access to resources                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           
+------- --------------------------------------------- ----------- -----------------------------------------
+access  {{internal.logins}}                           <all nodes> event:list,read,session:read,list                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
+auditor no-login-6566121f-b602-47f1-a118-c9c618ee5aec             session:list,read,event:list,read                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
+editor                                                            user:list,create,read,update,delete,...
+```
+
+The above output is from a fresh install of Teleport with the default roles, your cluster may have additional roles. In our example, let’s assume you want to give the bot `access` role to allow it to connect to SSH servers within your cluster.
+
+```bash
+$ tctl bots add robot --roles=access
+```
+
+## Step 3/4. Start Machine ID
+
+Now start Machine ID using the `tbot` binary.
```

Comment:
It also would be helpful to include common usage patterns with examples. For example, where do you normally start the tbot? I assume it's on the machine which will be using these certificates so e.g. your CI machine. Might be helpful to mention this explicitly.

Reply:
I figure there's (at least) 3 patterns to support here:

1. Demos and experimentation, where we run the bot manually as in the instructions here
2. One-shots, where we have the bot issue certs once and exit (pending support, planning to add a `--oneshot` flag for 9.0). Also expect this to be the use-case for IAM joining, CI, and so on.
3. Long-lived continuously-renewed certs, and ideally the bot is deployed in a production-hardened way. Dedicated system user account, systemd service, `tbot init` ACLs, etc.

Reply:
Good question, what is the purpose of the Getting Started guide? To outline a simple use case or enumerate all? I was thinking about later having different sections that explain the different modes?

Reply:
Different sections/guides/etc make sense to me, I think that mirrors our docs for the Teleport server as well.